packages/host/app/components/operator-mode/code-submode/playground/playground-preview.gts

```diff
@@ -64,7 +64,11 @@ const PlaygroundPreview: TemplateOnlyComponent<Signature> = <template>
       veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
       commodo consequat.</div>
   {{else if (eq @format 'fitted')}}
-    <FittedFormatGallery @card={{@card}} @isDarkMode={{true}} />
+    <FittedFormatGallery
+      @card={{@card}}
+      @isDarkMode={{true}}
+      @isField={{@isFieldDef}}
```

Comment:
is “field” conceptually the same as “fielddef”?

Reply:
the way the types are set, `FieldDef` means it's an instance of field definition. `typeof FieldDef` means the constructor. meaning of `isField` is vague, I can change that to be `isFieldDef`